source/docs/software/python/index.rst

```diff
@@ -0,0 +1,9 @@
+FRC Python Programming
+======================
+
+.. toctree::
+   :maxdepth: 1
+
+   installing-robotpy-on-roborio
+   deploying-robot-code
+   robot-simulator
```

Comment:
I'm not sure how I feel about this. It is probably better to merge into the existing articles, or at the very least, we should cross-reference.

In general, we should treat Python the exact same way as Java/C++ except for the mandatory differences

Reply:
For the simulator article, It seems like running dashboards with simulated robot code can be split off into it's own article instead of included multiple places.

Reply:
I find that the most problematic sections are that python cannot rely on the VS Code WPILib tools to upload and debug, but I'll see what can be done to merge everything.